### PR TITLE
Replace cdh with Collaborative Lesson Development Training

### PR DESCRIPTION
**Pull request by ndporter**
Replace link to the retired curriculum development handbook (which states): `The Carpentries is preparing to retire this handbook at the end of 2023. Existing content that has not already been replicated elsewhere will be relocated. See the relevant issue on the source repository for more details.`

Add link to current training.

### CODE REVIEW

**Comment by ndporter** [comment]
@ErinBecker or @tobyhodges please verify this is the correct link and approach. Thanks!

File: episodes/15-carpentries.md [hunk 1]
```diff
@@ -242,6 +242,7 @@ and a place to develop it in [The Carpentries Incubator][carpentries-incubator].
 Any lesson that uses The Carpentries [lesson template][lesson-template],
 follows our [Code of Conduct][coc], and is licensed either [CC-BY][cc-by] or [CC-0][cc-0]
 can be hosted in The Carpentries Incubator.
+
 Another resource, [The Carpentries Curriculum Development Handbook][cdh], provides a guide to
```
Comment (tobyhodges) [inline episodes/15-carpentries.md:246]:
```suggestion
Another resource, [the Collaborative Lesson Development Training curriculum][cld], provides a guide to
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Thanks @ndporter! I have suggested a change to the wording of the link.

**Comment by ndporter** [comment]
I tried to approve your changes (and did merge them in) @tobyhodges but it's still showing as "changes requested". Once that resolves either on its own or with coaxing, we should probably have someone else from @carpentries/instructor-training-maintainers check it and merge.

**Review by tobyhodges — APPROVED** [review]

**Comment by brownsarahm** [comment]
🎉

**Pull request merged**